tools/mpremote/mpremote/console.py

```diff
@@ -11,8 +11,13 @@
 class ConsolePosix:
     def __init__(self):
         self.infd = sys.stdin.fileno()
-        self.infile = sys.stdin.buffer.raw
-        self.outfile = sys.stdout.buffer.raw
+        self.infile = sys.stdin.buffer
+        self.outfile = sys.stdout.buffer
+        if hasattr(self.infile, "raw"):
+            self.infile = self.infile.raw
+        if hasattr(self.outfile, "raw"):
+            self.outfile = self.outfile.raw
```

Comment:
When running mpremote in the vscode terminal on OSX the sys.stdout.buffer did not have the raw attribute - it appeared to work fine without it.